javascript/ql/src/Declarations/UnreachableMethodOverloads.qhelp

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+    <p>
+        The TypeScript compiler has to choose which specific overload is called
+        when a method with multiple overloads is called.
+        The TypeScript compiler will always choose the textually first overload 
```

Comment:
I'd suggest either "The compiler will always..." or "It will always..." to avoid repetition with the first sentence

Reply:
I changed it to `The compiler will always...`